crates/tako/src/server/monitoring.rs

```diff
@@ -26,11 +26,11 @@ pub struct MonitoringEvent {
 pub enum MonitoringEventPayload {
     WorkerConnected(WorkerId, Box<WorkerConfiguration>),
     WorkerLost(WorkerId, LostWorkerReason),
-    OverviewUpdate(CollectedOverview),
+    OverviewUpdate(WorkerOverview),
 }
 
 // Keep the size of the event structure in check
-const _: () = assert!(std::mem::size_of::<MonitoringEventPayload>() == 32);
```

Comment:
Update the size for now, please (you will get a conflict after rebasing on `main` probably). In the future we will maybe box the overview to reduce the size of `MonitoringEventPayload`.